src/dsverifier.cpp

```diff
@@ -55,10 +55,18 @@
 #include <fstream>
 #include <streambuf>
 #include <math.h>
+#include <cassert>
```

Comment:
May I ask you to use C++ headers only? Can you also list them in alphabetic order?